src/text_engine/characters/PlayerCharacter.java

```diff
@@ -47,11 +51,11 @@ public PlayerCharacter(String name, String description, Room location) {
 
 
     @Override
-    protected int respondToInteraction(GameCharacter actor, GameEntity from, BufferedReader reader,
-                                       PrintWriter writer, String prompt) throws ExitToException {
-        Interactive choice = ConsoleActors.getChoice(reader, writer,
-                                                     actions.stream().map(f -> f.apply(this))
-                                                            .collect(Collectors.toList()), prompt);
+    protected int respondToInteraction(PlayerCharacter actor, GameEntity from, BufferedReader reader,
+                                       PrintWriter writer) throws ExitToException {
+        Interactive choice = ConsolePrompt.getChoice(
+                reader, writer, actions(actor, from, reader, writer),
+                Prompts.messages.getString("Prompts.genericPrompt"));
```

Comment:
IntelliJ can do autocomplete on all of these keys like `"Prompts.genericPrompt"` btw, which is real handy.